### PR TITLE
Fc

### PR DESCRIPTION
**Pull request by craigh**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | -
| Refs tickets      | -
| License           | MIT
| Changelog updated | no

## Description
update some code to be more FC to integration into master

### CODE REVIEW

File: src/system/SearchModule/Controller/UserController.php [hunk 1]
```diff
@@ -289,20 +288,19 @@ public function recentAction(Request $request)
...
 
-        $language = ZLanguage::getLanguageCode();
         $variableApi = $this->get('zikula_extensions_module.api.variable');
         $templateParameters = [
-            'siteName' => $variableApi->getSystemVar('sitename_' . $language, $variableApi->getSystemVar('sitename_en')),
-            'slogan' => $variableApi->getSystemVar('slogan_' . $language, $variableApi->getSystemVar('slogan_en')),
-            'metaKeywords' => $variableApi->getSystemVar('metakeywords_' . $language, $variableApi->getSystemVar('metakeywords_en')),
+            'siteName' => $variableApi->getSystemVar('sitename', $variableApi->getSystemVar('sitename_en')),
+            'slogan' => $variableApi->getSystemVar('slogan', $variableApi->getSystemVar('slogan_en')),
+            'metaKeywords' => $variableApi->getSystemVar('metakeywords', $variableApi->getSystemVar('metakeywords_en')),
```
Comment (cmfcmf) [inline src/system/SearchModule/Controller/UserController.php]: Did you remove the language intentionally?

File: src/system/ZAuthModule/Helper/MailHelper.php [hunk 2]
```diff
@@ -85,7 +85,7 @@ public function sendNotification($toAddress, $notificationType = '', array $temp
             $subject = $this->generateEmailSubject($notificationType, $templateArgs);
         }
 
-        $sitename = $this->variableApi->getSystemVar('sitename_' . \ZLanguage::getLanguageCode(), $this->variableApi->getSystemVar('sitename_en'));
+        $sitename = $this->variableApi->getSystemVar('sitename', $this->variableApi->getSystemVar('sitename_en'));
```
Comment (cmfcmf) [inline src/system/ZAuthModule/Helper/MailHelper.php]: You removed this everywhere. Maybe I don't understand?

File: src/system/SearchModule/Controller/UserController.php (hunk 1 above)
Comment (craigh) [inline src/system/SearchModule/Controller/UserController.php]: yes

**Comment by craigh** [comment]
> Also, I don't understand LocaleApi vs. LocaleMetadata. They seem to contain much duplicated code.

legacy vs. Core-2.0 Api.

the old classes will be deleted

File: src/system/ZAuthModule/Helper/MailHelper.php (hunk 2 above)
Comment (craigh) [inline src/system/ZAuthModule/Helper/MailHelper.php]: see https://github.com/zikula/core/pull/3086#issuecomment-249433022

**Comment by craigh** [comment]
hmm - possibly `LocaleMetadata` was an artifact of an earlier idea and needs to be removed.

**Comment by craigh** [comment]
that file is now removed.

**Pull request merged**